hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestPartitionStatsIndex.scala

```diff
@@ -178,7 +177,7 @@ class TestPartitionStatsIndex extends PartitionStatsIndexTestBase {
       override def apply(): Boolean = {
         try {
           doWriteAndValidateDataAndPartitionStats(hudiOpts,
-            operation = DataSourceWriteOptions.UPSERT_OPERATION_OPT_VAL,
+            operation = DataSourceWriteOptions.BULK_INSERT_OPERATION_OPT_VAL,
```

Comment:
This is to trigger the real concurrent writes; otherwise, one write fails due to conflicts.